tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -2248,13 +2248,18 @@ Duration <DURATION>"
 `;
 
 exports[`jsii_access_modifiers.test.w 1`] = `
-"error: Cannot access protected member \\"createTopic\\" of \\"Bucket\\"
+"error: Unknown symbol \\"createTopic\\"
```

Comment:
Regression

Reply:
It happened because I moved those to be internal:  https://github.com/winglang/wing/pull/5492#discussion_r1477217834
what do you think can be a better way?